test/integration/management-api/users.spec.ts

```diff
@@ -794,6 +793,47 @@ describe("users management API endpoint", () => {
         },
       ]);
     });
+
+    it("should return a list of users with totals", async () => {
+      const env = await getEnv();
+      const managementClient = testClient(managementApp, env);
+
+      const token = await getAdminToken();
+
+      // Now we should only get one result from the get endpoint but with nested identities
+      const response = await managementClient.api.v2.users.$get(
+        {
+          query: {
+            include_totals: "true",
+          },
+          header: {
+            "tenant-id": "tenantId",
+          },
+        },
+        {
+          headers: {
+            authorization: `Bearer ${token}`,
+          },
+        },
+      );
+
+      if (response.status !== 200) {
+        throw new Error(await response.text());
+      }
+
+      expect(response.status).toBe(200);
+
+      const body = await response.json();
+
+      if ("users" in body) {
+        expect(body.users.length).toBe(1);
+        expect(body.start).toBe(0);
+        expect(body.limit).toBe(10);
+        expect(body.length).toBe(1);
+      } else {
+        throw new Error("Expected an users property");
+      }
+    });
```

Comment:
I really wouldn't do this. We don't want any logic in our tests

The whole point of a testing framework is it does the errors in a standard way

Just expect `body.users` to exist and it will tell us if it doesn't

```suggestion
        expect(body.users!.length).toBe(1);
        expect(body.start).toBe(0);
        expect(body.limit).toBe(10);
        expect(body.length).toBe(1);

    });
```

Reply:
So, the problem with that solution is that it doesn't work :) 
![Uploading Screenshot 2024-05-03 at 20.13.56.png…]()

Reply:
I agree that it's not pretty, but honestly don't know how else to fix it.. 
This doesn't work: 
```
      expect(body).toHaveProperty("users");
      expect(body.users.length).toBe(1);
      expect(body.start).toBe(0);
      expect(body.limit).toBe(10);
      expect(body.length).toBe(1);
```

We could force typecast it but not sure if it's better?

This makes typescript happy and might be slightly prettier?
```
      if (!("users" in body)) {
        throw new Error("Expected an users property");
      }

      expect(body.users.length).toBe(1);
      expect(body.start).toBe(0);
      expect(body.limit).toBe(10);
      expect(body.length).toBe(1);

```